addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -231,13 +231,25 @@ def _l10n_it_post_invoices_step_2(self, invoices):
                 to_return[invoice] = {'error': self._format_error_message(_('The invoice has been refused by the Exchange System'), errors), 'blocking_level': 'error'}
                 invoice.l10n_it_edi_transaction = False
             elif state == 'notificaMancataConsegna':
-                to_return[invoice] = {
-                    'error': _('The E-invoice is not delivered to the addressee. The Exchange System is\
-                    unable to deliver the file to the Public Administration. The Exchange System will\
-                    contact the PA to report the problem and request that they provide a solution. \
-                    During the following 15 days, the Exchange System will try to forward the FatturaPA\
-                    file to the Administration in question again.'),
-                }
+                if invoice._is_commercial_partner_pa():
+                    to_return[invoice] = {'error': _(
+                        'The invoice has been issued, but the delivery to the Public Administration '
+                        ' has failed. The Exchange System will contact them to report the problem'
```

Comment:
```suggestion
                        'The invoice has been issued, but the delivery to the Public Administration'
                        ' has failed. The Exchange System will contact them to report the problem'
```

Not really important but you have 2 spaces :yum: